src/test/scala/dev/kamu/cli/MetadataRepositorySpec.scala

```diff
@@ -53,4 +58,56 @@ class MetadataRepositorySpec extends FunSuite with Matchers with KamuTestBase {
       actual3 shouldEqual Seq(dsC.id, dsB.id, dsA.id)
     }
   }
+
+  test(raw"'kamu add' from HTTP") {
+    withEmptyRepo { kamu =>
+      val testDir = getRandomDir()
```

Comment:
`withEmptyRepo` already creates a temporary directory which will be deleted after the tests (`kamu.config.repositoryRoot`), so you can any files in it and not bother deleting them. Currently if your test fails your directory will remain in `/tmp`.